Changelog.md

```diff
@@ -1,5 +1,25 @@
 ## Gazebo Fuel Tools 8.x
 
+### Gazebo Fuel Tools 8.0.2 (2023-03-30)
+
+1. Support link referral download
+    * [Pull request #333](https://github.com/gazebosim/gz-fuel-tools/pull/333)
+1. Allow Curl redirect on get.
```

Comment:
I know that this is copied and pasted from the merged PRs, but the changes in the PR add the curl redirect option for all methods. It might make sense to rename the PR and update this entry here.

Reply:
Done in [5d87b4f](https://github.com/gazebosim/gz-fuel-tools/pull/347/commits/5d87b4fad9abb69c6ae30022271e9e742e7c9ddf)